templates/default.yaml

```diff
@@ -123,6 +123,19 @@ mountInotify: null
 # ADVANCED CONFIGURATION
 # ===================================================================== #
 
+# User to be used inside the VM
```

Comment:
Can we have a test in https://github.com/lima-vm/lima/blob/master/hack/test-templates/test-misc.yaml